content/trusted-content/official-images/using.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Using Docker Official Images
+description: |
+  Learn about building applications with Docker Official images
+  and how to interpret the tag names they use.
+keywords: docker official images, doi, tags, slim, feedback, troubleshooting
+weight: 10
+---
+
+Docker recommends you use the Docker Official Images in your projects.
+These images have clear documentation, promote best practices, and are regularly updated.
+Docker Official Images support most common use cases, making them perfect for new Docker users.
+Advanced users can benefit from more specialized image variants as well as review Docker Official Images as part of your `Dockerfile` learning process.
+
+## Tags
+
+The repository description for each Docker Official Image contains a
+**Supported tags and respective Dockerfile links** section that lists all the
+current tags with links to the Dockerfiles that created the image with those
+tags. The purpose of this section is to show what image variants are available.
+
+![Example: supported tags for Ubuntu](../images/supported_tags.webp)
+
+Tags listed on the same line all refer to the same underlying image. Multiple
+tags can point to the same image. For example, in the previous screenshot taken
+from the `ubuntu` Docker Official Images repository, the tags `24.04`,
+`noble-20240225`, `noble`, and `devel` all refer to the same image.
+
+The `latest` tag for a Docker Official Image is often optimized for ease of use
+and includes a wide variety of useful software, such as developer and build tools.
+Because of their ease of use and wide applicability, `latest` images are often
+used in getting-started guides, but they're typically not optimized for production use.
+
+## Slim images
+
+A number of operating system and language stacks such as
+[Debian](https://hub.docker.com/_/debian/),
+[Node.js](https://hub.docker.com/_/node/),
+[Python](https://hub.docker.com/_/python/), and
+[Ruby](https://hub.docker.com/_/ruby/) have `slim` tag variants
+designed to provide a lightweight, production-ready base image
+with fewer packages.
```

Comment:
I think we should probably remove Debian from this list because the slim variants of Debian are not quite the same (even if they're in the same spirit) as these others -- they have the same packages, but less of the installed files from those packages (most notably, they exclude documentation not just from the packages in the image, but from all packages installed afterwards).

Reply:
Should we also remove "operating system and" from the lead in, since Debian and Oracle Linux are the only OS base image repos that offer slim variants, and both are similarly not recommended?